resources/user-agents/browsers/safari/safari-9-x.json

```diff
@@ -1,6 +1,6 @@
 {
   "division": "Safari #MAJORVER#.#MINORVER#",
-  "versions": ["9.1", 9, "8.1"],
```

Comment:
Removed 8.1 since it didn't show as being released in that wikipedia article (and no tests test for it).